test/Interop/Cxx/reference/reference-module-interface.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=Reference -I %S/Inputs -source-filename=x -enable-cxx-interop | %FileCheck %s
+
+// CHECK: func getStaticInt() -> Int32
+// CHECK: func getStaticIntRef() -> UnsafeMutablePointer<Int32>
+// CHECK: func getStaticIntRvalueRef() -> UnsafeMutablePointer<Int32>
+// CHECK: func getConstStaticIntRef() -> UnsafePointer<Int32>
+// CHECK: func getConstStaticIntRvalueRef() -> UnsafePointer<Int32>
+// CHECK: func setStaticInt(_: Int32)
+// CHECK: func setStaticIntRef(_: UnsafeMutablePointer<Int32>)
+// CHECK: func setStaticIntRvalueRef(_: UnsafeMutablePointer<Int32>)
```

Comment:
Could you file a bug about not creating or somehow resolving the ambiguity in imported Swift API in case `T&` and `T&&` are both in the same overload set?

For example, importing the following C++ API:

```c++
void example(int &);
void example(int &&);
```

should not make us create two identical Swift functions.

Reply:
Done - https://bugs.swift.org/browse/SR-12802.